src/sonic-host-services-data/debian/sonic-host-services-data.aaastatsd.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
How does service restart works, if it crashes

Reply:
Good point. So far, hostcfgd happens to be a *great* guy that we never got burnt by it. Thanks to all Try/Catch, I guess.

Said that, if aaaastatsd crashes, it may never come up until some config change related to it, as hostcfgd is less likely to crash.

So let us add auto-restart to it.

Reply:
ok